### PR TITLE
Implement basic test

### PR DESCRIPTION
**Pull request by mantinone**

### CODE REVIEW

File: test/basics.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const chai = require('chai');
+const expect = chai.expect;
+const testDummy = require('../src/TestDummy.js')
+
+describe( 'Are tests running', () => {
+
+  context ('When trying to run a basic test file', () => {
+      describe(' and trying to see if describe works', () => {
+        it('Should tell us tests are running', () => {
+          expect(true).to.eql(true)
```
Comment (deonna) [inline test/basics.js]: `eql` is for deep equality comparisons (e.g., if you're comparing an array of elements to another array of elements). Use `equal` here.

File: test/basics.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        })
+      })
+      describe('And testing functions from other files', () => {
+        it('Should say hello', () =>{
+          expect(testDummy.testfunction()).to.eql('Hello Mocha')
```
Comment (deonna) [inline test/basics.js]: `equal` - string comparisons aren't deep equality comparisons.

File: package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "description": "Dependent-Jaguar, MetroRail: Data Modeling & Database Design (LG Goal 126)",
   "main": "app.js",
   "scripts": {
-    "test": "mocha --watch dist/spec",
+    "test": "./node_modules/mocha/bin/_mocha",
```
Comment (deonna) [inline package.json]: Do you now have to run the `mocha` executable directly from the `node_modules` folder?

Might want to look into [why](http://stackoverflow.com/questions/9722407/how-do-you-install-and-run-mocha-the-node-js-testing-module-getting-mocha-co) if you have time.

**Review by deonna — APPROVED** [review]
Looks good so far. Make sure to read the Chai docs to know what assertions you should be using.

File: package.json (hunk 3 above)
Comment (mantinone) [reply package.json]: @deonna I spent a lot of time googling and looking at old projects to try and determine how to actually call my tests.  There's a lot of stuff going on in past projects that I don't understand but I never tried just directly using ' "test":"mocha" '!  That works now.  

Is there a different way I should be doing it, or is this good?

Comment (deonna) [reply package.json]: I think it's good for now. If you find yourself with a bunch of time at some point, we (or another coach) can try to debug it.

**Pull request merged**